doc/ext-pake/proposed/spake2plus_proposal_draft.md

```diff
@@ -62,3 +62,102 @@ Key confirmation is part of the SPAKE2+ protocol. Current PSA Cryptography API 1
   psa_pake_get_explicit_key(psa_pake_operation_t *pake, psa_key_attributes_t *attributes, psa_key_id_t *explicit_key);
   ```
 
+### PSA PAKE API Flow for SPAKE2+ with Matter as example
```

Comment:
👍 This is helpful to have here (and probably in the spec in some form) is helpful to think through the API design issues.